aruba/features/format/jks.feature

```diff
@@ -0,0 +1,169 @@
+Feature: JKS format output
+
+  As user, I need VCert to output my certificate, private key, and chain certificates in the JKS format
+  required by my application so that I don't have to use OpenSSL to combine the individual PEM files generated
+  by VCert into a PKCS#12 keystore and then use the Java KeyTool to convert the PKCS#12 keystore to a Java keystore.
+
+  - User requests JKS by specifying "jks" after the -format switch
+
+  - User must use the -file switch to specify the name of the keystore file when they specify -format jks
+    (i.e. neither the -cert-file, -key-file, nor -chain-file switches may appear on the command line,
+    and console output as a base64 encoded blob will not be supported).
+
+  - User always will be request for a password of at least 6 characters when he request a JKS file so it isn't possible to use the -no-prompt switch
+
+  - User specifies the password for the JKS file and the key entry, conformed by the private kye, the certificate and the chain certificates, using, preferable,
+    the -jks-password or alternatively the -key-password switch
+
+  - User must use the -jks-alias switch to provide the alias for the jks entry that will conformed by the private kye, the certificate and the chain certificates
```

Comment:
Minor typo at "jks entry that will conformed by"...it must be "jks entry that will be conformed by "

Reply:
Also typo o "kye" word. Must be "key"

Reply:
fixing